src/molecules/VideoCard/VideoCard.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import VideoCard from './VideoCard';
+
+describe('Video Card works properly', () => {
+  it('contains an image', () => {
+    render(
+      <VideoCard
+        id="test-card"
+        thumbnail="someURL"
+        subheader="Test"
+        description="This is a description"
+      />
+    );
+    expect(screen.getByTestId('test-card').firstChild).toHaveStyle(
```

Comment:
Instead of targeting the whole card with its `data-testid` I would suggest you to add `data-testid` for the part you want to test. This way you are sure that you are targeting the specific portion of that component. Some for the ones below.